src/plugins/hover/components.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import compose from 'recompose/compose';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import components from '../../components';
+import { rowMouseEnter, rowMouseLeave } from './actions';
+
+export const RowContainer = compose(
+    connect(
+        state => {
+            return {
+                hoveredRowKey: state.get('hoveredRowKey', null)
```

Comment:
I'm not entirely convinced it should be the plugin's responsibility to inject `hoveredRowKey` into `Cell` (or anywhere). It seems to be an implementation detail of this demo. I'm more inclined to avoid customizing `Row` altogether, letting `ButtonsColumn` be connected instead:

```javascript
const ButtonsColumn = connect(
    state => ({ hoveredRowKey: state.get('hoveredRowKey', null) })
  )(
    ({griddleKey,hoveredRowKey}) =>
      (griddleKey === hoveredRowKey) && <button>Edit</button> : null
  );
```